### PR TITLE
bump cosmos-sdk version to 0.37.13

### PR DESCRIPTION
**Pull request by tac0turtle**
Closes: #XXX

## Description

Bump sdk version to 0.37.13

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Review by fedekunze — APPROVED** [review]
utACK

File: go.sum [hunk 1]
```diff
@@ -274,6 +278,8 @@ github.com/tendermint/iavl v0.12.4/go.mod h1:8LHakzt8/0G3/I8FUU0ReNx98S/EP6eyPJk
 github.com/tendermint/tendermint v0.32.1/go.mod h1:jmPDAKuNkev9793/ivn/fTBnfpA9mGBww8MPRNPNxnU=
 github.com/tendermint/tendermint v0.32.11 h1:o770nBHgDPnANsMXG5bMR5ibMQYB14PFtUum64UA/Uc=
 github.com/tendermint/tendermint v0.32.11/go.mod h1:5/B1XZjNYtVBso8o1l/Eg4A0Mhu42lDcmftoQl95j/E=
+github.com/tendermint/tendermint v0.32.12 h1:JFD3aQZFySamBeU1bfDaqOVgFo6w8UpmiKylVB8eV00=
```
Comment (alessio) [inline go.sum:275]: Please run `go mod tidy`

File: Dockerfile [hunk 2]
```diff
@@ -5,7 +5,7 @@
 FROM golang:alpine AS build-env
 
 # Set up dependencies
-ENV PACKAGES python3 curl make git libc-dev bash gcc linux-headers eudev-dev
+ENV PACKAGES curl make git libc-dev bash gcc linux-headers eudev-dev python3
```
Comment (alessio) [inline Dockerfile:8]: Not a big expert of alpine's package manager here, but how does this affect the build of the image? It seems like that changing the order of the packages matters

**Review by alessio — CHANGES_REQUESTED** [review]
One change + request for clarification

File: Dockerfile (hunk 2 above)
Comment (tac0turtle) [reply Dockerfile:8]: python is deprecated. pythong2 has officially been deprecated

Comment (alessio) [reply Dockerfile:8]: Yes, I know :) It just looked weird that the order of the packages mattered. Anyways, nevermind :+1:

**Review by alessio — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (tac0turtle) [reply Dockerfile:8]: oh no this was my mistake. I accidentally pushed python3 before curl directly to rc1, so this commit moved it back to where it was before

**Pull request merged**